docs/api/index.rst

```diff
@@ -5,12 +5,6 @@ This section of the documentation details the public API
 usable to get details of projects, builds, versions and other details
 from Read the Docs.
 
-.. warning::
-
-    Originally, the Read the Docs API allowed connections over insecure HTTP.
-    Starting in January 2019, requests over HTTP
-    will be automatically redirected to HTTPS
-    and non-GET/HEAD requests over insecure HTTP will fail.
```

Comment:
I removed this while I was here, but technically this change is not due to API v1 but rather due to https://github.com/rtfd/readthedocs-ops/pull/461.